sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Airflow DAG run instances
+description: List of DAG run instances
+labels:
+  incremental: false
+owners:
+  - kignasiak@mozilla.com
+scheduling:
+  dag_name: bqetl_monitoring_airflow
+  referenced_tables:
+    - ['moz-fx-data-bq-fivetran', 'airflow_metadata_airflow_db', 'dag_run']
```

Comment:
There is also a configuration option to trigger the Airflow import in this DAG: https://github.com/mozilla/bigquery-etl/blob/f6b20988b5f19c9ce5ce04eda405a7bae89e6ff7/docs/reference/scheduling.md?plain=1#L64-L67

Reply:
made that tweak.